### PR TITLE
feat: add tooltip scenario

### LINKED ISSUE #23: Tooltips
Scenario that generates tooltips

### PR DESCRIPTION
**Pull request by e-taniguti**
### Description
New scenario covering tooltips on different element types

### Related Issue
Closes #23 

### Checklist

- [x] I have read the contribution guidelines.
- [x] This PR includes tests (or updates existing tests) that cover the changes.
- [ ] Documentation has been updated (if applicable).
- [x] Code follows the project's coding standards.
- [x] All tests pass.

### Screenshots (if applicable)
Screenshot 2023-11-10 at 11 43 42
Screenshot 2023-11-10 at 11 43 36
Screenshot 2023-11-10 at 11 43 31

### CODE REVIEW

File: src/scenarios/Tooltip.scenario.tsx [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+            <IconButton disabled>
+              <Delete />
+            </IconButton>
+          </span>
+        </Tooltip>
+
+        <HtmlTooltip
+          placement='right'
+          title={
+            <React.Fragment>
```
Comment (gregoryduckworth) [inline src/scenarios/Tooltip.scenario.tsx:61]: The `React.Fragment` isn't necessary here, you could just wrap it in `<>...</>`.

File: src/tests/Tooltip.test.tsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+import TestRenderer from './TestRenderer'
+import TooltipPage from '../scenarios/Tooltip.scenario'
+
+describe('CheckBox component', () => {
+  beforeEach(() => {
+    sessionStorage.clear()
+  })
+
+  it('renders tootip when overing disabled button', async () => {
```
Comment (gregoryduckworth) [inline src/tests/Tooltip.test.tsx:14]: Just a spelling mistake here:
```suggestion
  it('renders tootip when hovering disabled button', async () => {
```
and the same is needed on the below.

**Review by gregoryduckworth — COMMENTED** [review]
Just a few small tweaks but looks good 👍

File: src/scenarios/Tooltip.scenario.tsx [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+          placement='right'
+        >
+          <span data-testid='delete-button'>
+            <IconButton disabled>
+              <Delete />
+            </IconButton>
+          </span>
+        </Tooltip>
+
+        <HtmlTooltip
```
[the remark this answers is not in the document]
Comment (e-taniguti) [reply src/scenarios/Tooltip.scenario.tsx]: nice catch !!!
removed.

File: src/scenarios/Tooltip.scenario.tsx [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+              </Link>
+            </React.Fragment>
+          }
+        >
+          <Button variant='outlined' data-testid='more-info-button'>
+            {t('scenarios.tooltip-page.more-information')}
+          </Button>
+        </HtmlTooltip>
+
+        <div
```
[the remark this answers is not in the document]
Comment (e-taniguti) [reply src/scenarios/Tooltip.scenario.tsx]: changes applied

File: src/scenarios/Tooltip.scenario.tsx (hunk 1 above)
Comment (e-taniguti) [reply src/scenarios/Tooltip.scenario.tsx]: Changes applied

File: src/tests/Tooltip.test.tsx (hunk 2 above)
Comment (e-taniguti) [reply src/tests/Tooltip.test.tsx]: Typo fixed

**Review by gregoryduckworth — APPROVED** [review]
🚀

**Pull request merged**